udata/core/dataset/apiv2.py

```diff
@@ -73,7 +73,7 @@
 log = logging.getLogger(__name__)
 
 ns = apiv2.namespace("datasets", "Dataset related operations")
-search_parser = DatasetSearch.as_request_parser()
+search_parser = DatasetSearch.as_request_parser(store_missing=False)
```

Comment:
We use `store_missing=False` because we don't want to send filters set to `None` to the search service (Mongo does not care).